src/server.js

```diff
@@ -88,14 +154,113 @@ export class Server {
+          'base64'
+        );
+
+        const results = responseXDR.result().value();
+
+        let offerResults;
+        let hasManageOffer;
+
+        if (results.length) {
+          offerResults = results
+            .map((result, i) => {
+              if (typeof result.value().value().success !== 'function') {
+                return null;
+              }
+
+              hasManageOffer = true;
+
+              let amountBought = new BigNumber(0);
+              let amountSold = new BigNumber(0);
+
+              const offerSuccess = result
+                .value()
+                .value()
+                .success();
+
+              const offersClaimed = offerSuccess
+                .offersClaimed()
+                .map((offerClaimed) => {
+                  const claimedOfferAmountBought = new BigNumber(
+                    // amountBought is a js-xdr hyper
+                    offerClaimed.amountBought().toString()
+                  );
+                  const claimedOfferAmountSold = new BigNumber(
+                    // amountBought is a js-xdr hyper
+                    offerClaimed.amountSold().toString()
+                  );
+
+                  // This is an offer that was filled by the one just submitted.
+                  // So this offer has an _opposite_ bought/sold frame of ref
+                  // than from what we just submitted!
+                  // So add this claimed offer's bought to the SOLD count and vice v
+
+                  amountBought = amountBought.add(claimedOfferAmountSold);
+                  amountSold = amountSold.add(claimedOfferAmountBought);
+
+                  return {
+                    sellerId: offerClaimed
+                      .sellerId()
+                      .value()
+                      .toString('hex'),
+                    offerId: offerClaimed.offerId,
+                    assetSold: offerClaimed.assetSold().switch(),
+                    amountSold: claimedOfferAmountSold
+                      .div(STROOPS_IN_LUMEN)
+                      .toString(),
+                    assetBought: offerClaimed.assetBought().switch(),
+                    amountBought: claimedOfferAmountBought
+                      .div(STROOPS_IN_LUMEN)
+                      .toString()
+                  };
+                });
+
+              const effect = offerSuccess.offer().switch().name;
+
+              return {
+                offersClaimed,
+                effect,
+                index: i,
+                lastOfferId: offersClaimed.reduce(
+                  (memo, offerClaimed) => offerClaimed.offerId,
+                  null
+                ),
+                // this value is in stroops so divide it out
+                amountBought: amountBought.div(STROOPS_IN_LUMEN).toString(),
+                amountSold: amountSold.div(STROOPS_IN_LUMEN).toString(),
+                isFullyOpen:
+                  !offersClaimed.length && effect === 'manageOfferCreated',
```

Comment:
Shouldn't it be something like `... && effect != 'manageOfferDeleted'`? To me, an updated offer could still be "fully open" based on your description of this field above.